ci/code_checks.sh

```diff
@@ -102,9 +102,17 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
 
     MSG='Check for use of not concatenated strings' ; echo $MSG
     if [[ "$GITHUB_ACTIONS" == "true" ]]; then
-        $BASE_DIR/scripts/validate_string_concatenation.py --format="[error]{source_path}:{line_number}:{msg}" .
+        $BASE_DIR/scripts/validate_string_concatenation.py --validation-type="strings_to_concatenate" --format="##[error]{source_path}:{line_number}:{msg}" .
```

Comment:
out of curiosity, what does the `##` here mean?

Reply:
Full disclosure I have no clue, I asked @datapythonista for the output format that it should be outputted when running on ```github-actions```.

Reply:
Just a convention in GitHub Actions. Any row in the output that starts with `##[error]` is highlighted as an error, and it's easier to find in the logs.